tests/e2e/api-core-tests/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Editors
```

Comment:
These should be cross referenced with the ignores in the repository root and have duplicates removed.

Reply:
Thanks! I removed all of the duplicates aside from a few, such as `node_modules`, the `.env` file and the `collection.json` output. We have [the same duplicate entries in the API package `.gitignore`](https://github.com/woocommerce/woocommerce/blob/trunk/tests/e2e/api/.gitignore) (just noting this as this is where I copied most of this over from), should we also look at removing those in a future update?

Reply:
Created https://github.com/woocommerce/woocommerce/issues/30655.